packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -289,6 +297,7 @@ function SingleForm({ form, appUrl, Page }: SingleFormComponentProps) {
                 <div className="lg:min-w-72 lg:max-w-72 mb-6 md:mr-6">
                   <TextField
                     type="text"
+                    label="Routing forms"
```

Comment:
Again confused here. Why are we labelling a field as Routing Forms when it is the name of the Routing Form you are editing.